FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -104,38 +105,34 @@ TrajectorySeedProducer::TrajectorySeedProducer(const edm::ParameterSet& conf):
 bool
 TrajectorySeedProducer::pass2HitsCuts(const TrajectorySeedHitCandidate & innerHit,const TrajectorySeedHitCandidate & outerHit) const
 {
-
-    const DetLayer * innerLayer = 
-	measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(innerHit.hit()->det()->geographicalId());
-    const DetLayer * outerLayer = 
-	measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(outerHit.hit()->det()->geographicalId());
+  const DetLayer * innerLayer =
+    measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(innerHit.hit()->det()->geographicalId());
+  const DetLayer * outerLayer =
+    measurementTrackerEvent->measurementTracker().geometricSearchTracker()->detLayer(outerHit.hit()->det()->geographicalId());
+  vector<BaseTrackerRecHit const *> innerHits;
```

Comment:
write std::vector
and to avoid the push_back, use other constructor
std::vector<BaseTrackerRecHit const *> innerHits(1,(BaseTrackerRecHit*)innerHit)
(same for outerhit)